docs/pipeline-components-and-applications/loaders-storage-targets/google-cloud-storage-loader/index.md

```diff
@@ -14,63 +14,38 @@ import CodeBlock from '@theme/CodeBlock';
 
 Cloud Storage Loader is a [Dataflow](https://cloud.google.com/dataflow/) job which dumps event from an input [PubSub](https://cloud.google.com/pubsub/) subscription into a [Cloud Storage](https://cloud.google.com/storage/) bucket.
 
-## Technical details
-
 Cloud Storage loader is built on top of [Apache Beam](https://beam.apache.org/) and its Scala wrapper [SCIO](https://github.com/spotify/scio).
 
-It groups messages from an input PubSub subscription into configurable windows and write them out to Google Cloud Storage.
-
-It also optionally partitions the output by date so that you can easily see what was outputted when in your cloud storage bucket, for example we could see the following hierarchy:
-
-```bash
```

Comment:
This particular example seems useful, could we include it as an illustration of the partitioning configuration option below?